src/containers/markericon.jsx

```diff
@@ -7,61 +7,49 @@ import ICONS from './icons';
 // Using L because a component returns React.createElement() instead of L.geoJSON(..., { pointToLayer }
 // Reference: https://github.com/PaulLeCam/react-leaflet/issues/234
 
-const setMarker = (properties, latlng) => {
-  const orderType = {
-    Basilian: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Augustinian Canons': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Knights of the Hospital of Saint John of Jerusalem': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Benedictine: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Cistercian: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Cluniac: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Knights Templar': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(
-          <Icon icon={ICONS.MONASTERY} translate="-6,-41" scale="1,-1" />
-        ),
-      }),
+const orderLookup = order => {
+  switch (order) {
+    case 'Basilian':
+      return { color: 'blue', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Augustinian Canons':
+      return { color: 'red', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Knights of the Hospital of Saint John of Jerusalem':
+      return { color: 'green', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Benedictine':
+      return { color: 'yellow', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Cistercian':
+      return { color: 'purple', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Cluniac':
+      return { color: 'orange', scale: '1,-1', translate: '-6,-41' };
+
+    case 'Knights Templar':
+      return { color: 'brown', scale: '1,-1', translate: '-6,-41' };
+
+    default: {
```

Comment:
This case isn't necessary since we will initialize the object with the `defaultProps` in the call to `Object.assign`

Reply:
Done